action.yml

```diff
@@ -48,7 +48,8 @@ runs:
         fi
         if [[ $INPUT_FIX_FORMAT != true ]]; then
           echo
-          echo ::error::Terraform format is broken, see diff for details. To fix, run: terraform fmt -recursive
+          echo ::error::Terraform formatting errors detected. See log for diff with detail.
+          echo ::error::To fix formatting, run locally: terraform fmt -recursive
```

Comment:
I think a single error block looks better

Reply:
I agree, kinda knew that when I pushed! Fixed.